lib/Reactify.re

```diff
@@ -4,23 +4,22 @@ module Make = (ReconcilerImpl: Reconciler) => {
   /* Module to give us unique IDs for components */
   type renderedElement =
     | RenderedPrimitive(ReconcilerImpl.node)
-  and elementWithChildren = (childComponents, Effects.effects, Context.t)
+  and elementWithChildren = (list(element), Effects.effects, Context.t)
   and render = unit => elementWithChildren
-  and component =
+  and element =
```

Comment:
Thanks for streamlining the nomenclature - `element` and `instance` is better and simplified - less concepts to juggle. And the rename of the `component` function to `createComponent` helps a lot, too!